### PR TITLE
ISSUE #4830 - 2d Milestone 1 

### LINKED ISSUE #4380: Issue adding new users as admin
### Description

When inviting new users to a TeamSpace the invite button is disabled when assigning an admin. There is a workaround to add a different permission level and then go back to admin 

- This is happening in: both 
- This requires a hotfix to production: yes 

### Steps to replicate
- Open this model: any model
- try add a user that does not exist with admin permissions 
- when not found click sned invite 
-  the send invite button will be diabled for no reason 
- you can then slect different permissions and back to amdin to make it work

### Current Behaviour
- Send invite is disabled 

### Expected Behaviour
- send invite should be available when all fields complete 

### Pictures/ Gifs of the error
bug sign up

### LINKED ISSUE #4830: [2D MVP] Milestone 1
### Description
This is the base branch for 2d milestone 1 
dependencies:
- [x] https://github.com/3drepo/3drepo.io/issues/4781
- [x] https://github.com/3drepo/3drepo.io/issues/4779
- [x] https://github.com/3drepo/3drepo.io/issues/4780
- [x] https://github.com/3drepo/3drepo.io/issues/4788
- [x] https://github.com/3drepo/3drepo.io/issues/4789
- [x] https://github.com/3drepo/3drepo.io/issues/4782
- [x] https://github.com/3drepo/3drepo.io/issues/5023
- [x] https://github.com/3drepo/3drepo.io/issues/5024
- [x] https://github.com/3drepo/3drepo.io/issues/4985
- [x] https://github.com/3drepo/3drepo.io/issues/5080
- [x] https://github.com/3drepo/3drepo.io/issues/4826
- [x] https://github.com/3drepo/3drepo.io/issues/4898
- [x] https://github.com/3drepo/3drepo.io/issues/4901
- [x] https://github.com/3drepo/3drepo.io/issues/4983
- [x] https://github.com/3drepo/3drepo.io/issues/5126
- [x] https://github.com/3drepo/3drepo.io/issues/5127
- [x] https://github.com/3drepo/3drepo.io/issues/5128
- [x] https://github.com/3drepo/3drepo.io/issues/5129

### Goals
- [x] As an Admin I want to Upload/delete their Drawings
- [x] As a user with access to the drawing I want to view the revision history of the drawings
- [x] As an admin I want to manage permissions on each drawing.

### Related Resources
[MockUp](https://www.figma.com/file/uTWnaYZ8IyoTn2i2xKtdBo/2D---3D?type=design&node-id=5-5981&mode=design&t=ATm59VQXYC23fw9g-0)

### PR DESCRIPTION
**Pull request by sanmont3drepo**
This fixes #4830 

#### Description
- Milestone 1  for 2d drawing. More details [here](https://github.com/3drepo/3D-Repo-Product-Team/issues/377)

### CODE REVIEW

File: backend/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (carmenfan) [inline backend/.vscode/launch.json:1]: The sizing logic here is a bit odd (this is half of a 1080 screen):
image

Comment (carmenfan) [reply backend/.vscode/launch.json:1]: Behaviour also seems to change  for no clear reason depending on whether I upload a pdf or dwg, the drawing side panel only pops up automatically if I upload a pdf:

**Review by carmenfan — CHANGES_REQUESTED** [review]

File: backend/.vscode/launch.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (sanmont3drepo) [reply backend/.vscode/launch.json:1]: moved into #5127

[the remark this answers is not in the document]
Comment (sanmont3drepo) [reply backend/.vscode/launch.json:1]: Moved into #5128

File: backend/.vscode/launch.json
Comment (carmenfan) [inline backend/.vscode/launch.json:1]: image

Tooltip doesn't give correct information on drawings (it expands/collapse the revision list, it doesn't launch the drawing)

**Review by carmenfan — DISMISSED** [review]
Generally fine just a small tooltip that needs fixing

**Review by carmenfan — APPROVED** [review]

**Pull request merged**